### ISSUE TITLE
Add ability to recreate a container with the supervisor api

### ISSUE THREAD

**Issue by CameronDiver** [balena-io/balena-supervisor#919]
Maybe this can be added as an argument to the restart endpoint.

**Comment by openedhardware** [comment]
@CameronDiver 
When do you expect to add this feature?

I am running multiple containers on a single RPi, and need to start a new container during the runtime...

Cheers.

**Comment by CameronDiver** [comment]
Hey @openedhardware do you need to create a container which is not referenced in your docker-compose? Or do you need to start a stopped container from another container? This issue references the ability to recreate an already running container.

**Comment by openedhardware** [comment]
@CameronDiver 
Yeah, I want to create a container that is not described in my `docker-compose`.
Any idea how to do that?

**Comment by CameronDiver** [comment]
@openedhardware apologies for the delay here, I missed the email!

For this you can include the balena engine socket into your project, using the `io.balena.features.balena-socket` label, described here: https://www.balena.io/docs/learn/develop/multicontainer/#labels

Using this, you can point a docker CLI or SDK at the socket and perform actions upon the engine (such as downloading and starting another container). Let me know if you'd like further details.
